.github/workflows/linux_builds.yml

```diff
@@ -126,17 +126,47 @@ jobs:
           scons --version
 
       # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
+      # [Workaround] SwiftShader have problems with tesselation, so we comment this for now
```

Comment:
Is there an upstream bug report about this, so we can track resolution?

Reply:
> [Workaround] SwiftShader have problems with tesselation, so we comment this for now

Tessellation is [not implemented at all](https://github.com/google/swiftshader/blob/7c84426d2abe231f3087a2942fd831137eb3b0aa/src/Vulkan/VkPhysicalDevice.cpp#L388-L400), and probably software implementation is not feasible for performance reasons.

> Is there an upstream bug report about this, so we can track resolution?

There's feature proposal in the SwiftShader bug tracker [Implement tesselation support [147805271]](https://issuetracker.google.com/issues/147805271), but it's for ANGLE/OpenGL ES, and there's no activity.

Reply:
We should probably change Godot code to handle missing tessellation gracefully then if we want to use SwiftShader (and let others use it).
This `sed` workaround is not really sustainable in the long run.